docs/architecture_documentation/project_structure.rst

```diff
@@ -1,26 +1,99 @@
-Project structure
+Project Structure
 #####################
 
+This site gives a brief overview of the project structure of the qunicorn project.
+Describing the different packages and components within.
+
+The Structure described can be found within the **qunicorn_core** package.
+
 Structure of the project
 *************************
 
-api:
-* api_models package: DTOs and Schemas for all Objects
-   * these are used for computation and to define the schemas for the api views
-* {object}_api packages:
-   * these include the definition of the different views
-
-core:
-* mapper package:
-   * this includes different mappers from DTOs to Database Objects. A Helper class.
-* {object}manager packages:
-   * Computational classes: exist for different objects
-   * Service classes: get called from api
-       * All calls from API should be forwarded to these classes.
-       * Celery tasks
-
-db:
-* database_services package:
-   * services to access the database (add, get, update, remove)
-* models package:
-    * definition of various database models
+api <Package>:
+^^^^^^^^^^^^^^^^
+
+The api package contains all the api views and models. It is used to define the API.
+
+* api_models <Package>:
+    * contains DTOS and Schemas for all Objects
+    * these are used for computation and to define the schemas for the api views
+* deployment_api <Package>:
```

Comment:
Not sure if all that is necessary, maybe summarizing the "XXX_api" files and writing a generic description is sufficient. (less redundancy and more clearness)